platform/ui/src/utils/getFormattedDate.js

```diff
@@ -0,0 +1,12 @@
+import moment from 'moment';
```

Comment:
Since `@ohif/ui` already have `momentjs` as a dependency and to avoid having the same dependency around other projects, I ended up creating a new `utils` that lives under the UI project. LMK what you guys think about it